### ISSUE TITLE
The harvester service exits without any error

### ISSUE THREAD

**Issue by ageorge95** [gowke/lotus-blockchain#9]
As the title states, when using the latest commit available: 9f4bc673418e9173905a22d335df0fc97b930d55, the harvester service starts but then it exits after a couple of seconds.

There are no errors in the log files nor in the CLI instance that started it.

- Also, if I choose to start it ("lotus start harvester") it says that it is already running. If I try to restart it it says that the service was closed and then opened, but I see no actual harvester service running
- If there are no plot directories configured then the harvester remains opened
- The behavior described above only happens if I add any plot directories

Is this happening only on my side ?

**Comment by gowke** [comment]
try killing all lotus processes or running lotus stop all -d 
I dont know what is the farmer/harvester setup you are running. 
There could be a clash with a few coins on rpc ports, you could change those in the settings though. (I think sector was one of them)

**Comment by ageorge95** [comment]
Thanks for the response 👍

I tried all that but still I could not get the harvester to remain open.

I am farming all the forks (that still have a working timelord), and this is the first time I see this issue.

So, you are confirming that on your side 9f4bc673418e9173905a22d335df0fc97b930d55 works as intended right ?

If yes, then at least I know that it is something from my side.

**Comment by gowke** [comment]
Yes working as intended, no issues really, been reinstalling it myself to double check. its all good. 

Also please check out lotusgang.org.
